Python/basicrequest.py

```diff
@@ -133,13 +133,14 @@ def putMessage():
         queue.put(EnvStore.stringForVersion(json.dumps(msgObject)), md)
         # queue.put(str(json.dumps(msgObject)), md)
         # queue.put("Hello")
-
+        qmgr.commit()
```

Comment:
you have a commit and rollback of the request, but you haven't set put message options of SYNCPOINT.